src/marks/delaunay.js

```diff
@@ -279,9 +279,9 @@ function voronoiof(delaunay, dimensions) {
   return delaunay.voronoi([marginLeft, marginTop, width - marginRight, height - marginBottom]);
 }
 
-function delaunayMark(DelaunayMark, data, {x, y, ...options} = {}) {
+function delaunayMark(DelaunayMark, data, {x, y, initializer, ...options} = {}) {
   [x, y] = maybeTuple(x, y);
-  return new DelaunayMark(data, {...options, x, y});
+  return new DelaunayMark(data, {...basic({...options, x, y}, exclusiveFacets), initializer});
```

Comment:
Is `exclusiveFacets` only needed in the `Voronoi` case, not the general case of all Delaunay marks?

Reply:
Yes I think so. The hand-wavy "proof" is that the voronoi cells are "more" associated to each data point, and can be separated by facet after the transform, whereas the delaunay triangles (and mesh) are tied to the whole group of points, and can't be separated post-transform (any triangle that is not joining 3 points in the same facet would be inconsistent).